src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/PasswordChangeResponse.cs

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.ServiceLayer.Connection.Contracts
+{
+    /// <summary>
+    /// Parameters to be sent back after a password change attempt.
+    /// </summary>
+    public class PasswordChangeResponse
+    {
+        /// <summary>
+        /// Status indicating if connection was successful or not.     
+        /// </summary>
+        public bool Result { get; set;  }
+
+        /// <summary>
+        /// Additional optional detailed error messages, if an error occurred.
+        /// </summary>
+        public string? Messages { get; set; }
```

Comment:
how is this used in the client side?

Reply:
This is used to return the header of the error message (simplified description of the error), while the ErrorMessage returns the full error message, I should probably rename this.

Reply:
I mean in the UI, do you differentiate between ErrorMessage and Messages? In my opinion you can simplify combine them together and show it in the error dialog.

Reply:
Yeah, they are differentiated. So no change here.

Reply:
how? can you share a screenshot?

Reply:
<img width="481" alt="Password Change Dialog password cannot be changed" src="https://user-images.githubusercontent.com/18018452/204912492-f1372aa9-5f46-47c4-a543-5b1ae29aa4e1.png">
<img width="482" alt="Password Change Dialog Empty Password" src="https://user-images.githubusercontent.com/18018452/204912677-c81b73e6-be38-4203-8a7e-e093ed4b548b.png">

The message at the top is the ErrorMessage (description), and the ErrorDetails are in the middle. If this is unnecessary, please let me know.

Reply:
ok, based on how it is used, you just need one property.